playbook/app/pb_kits/playbook/pb_dialog/dialog.html.erb

```diff
@@ -5,38 +5,38 @@
         id: object.id,
         class: object.classname) do %>
             <% if object.status === "" && object.title %>
-                <%= pb_rails("dialog/dialog_header", props: {title: object.title, id: object.id }) %>
+                <%= pb_rails("dialog/dialog_header", props: { title: object.title, id: object.id }) %>
             <% end %>
             <% if object.status === "" && object.text %>
                 <%= pb_rails("dialog/dialog_body", props: { text: object.text }) %>
             <% end %>
             <% if object.status != "" %>
-                <%= pb_rails("dialog/dialog_body", props: {padding: "md"}) do %>
-                    <%= pb_rails("flex", props: {align: "center", orientation: "column"}) do %>
+                <%= pb_rails("dialog/dialog_body", props: { classname: "dialog_status_text_align" ,padding: "md" }) do %>
```

Comment:
Why are we only adding this for this specific use case (this one conditional)? 
Does it not make sense to add when the user also passes in text?
 
I reached out to a designer just to confirm that we are indeed wanting to change this completely. We normally would want to cut an alpha for something like this that changes default styling on a kit. (we are changing the way it looks on people's actual UI in nitro/other clients).